<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/tray_icon.ico</normaloff>:/images/tray_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Overview</string>
     </attribute>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>421</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="openDialog">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font-weight:bold;</string>
      </property>
      <property name="text">
       <string>Change Interactivity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Directory View :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createNewInter">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font-weight:bold;</string>
      </property>
      <property name="text">
       <string>Create New Interactivity</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>JS</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>JS Files List :</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>0</y>
        <width>16</width>
        <height>401</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_jsProps">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>421</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>JS Properties</string>
      </property>
      <widget class="QCheckBox" name="jsViewNextLoadCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>161</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>isNextStepLoad</string>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="jsViewList">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>30</y>
        <width>271</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Templates</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Templates List:</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>283</x>
        <y>0</y>
        <width>20</width>
        <height>401</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_templates">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>421</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Template Properties </string>
      </property>
      <widget class="QCheckBox" name="templateNextLoadCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>131</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>isNextStepLoad</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QListWidget" name="templateList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>271</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>CSS</string>
     </attribute>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>20</width>
        <height>401</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="cssList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>271</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CSS List :</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_CSS">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>421</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string> CSS  Properties  </string>
      </property>
      <widget class="QCheckBox" name="cssNextLoadCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>131</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>isNextStepLoad</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Media</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Loc-acc Json</string>
     </attribute>
     <widget class="QTreeWidget" name="locTreeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>341</width>
        <height>311</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">Screen</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>353</x>
        <y>0</y>
        <width>20</width>
        <height>401</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_addScreen">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>391</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Add Screen</string>
      </property>
      <widget class="QLineEdit" name="screenNameText">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="addScreenBtn">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Screen</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ScreenId :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="screenNameText_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ScreenName  :</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_addElement">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>391</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Add Element</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ElementId :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AccId :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addEleBtn">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>140</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Element</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="eleNameText">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="eleAccIdText">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Role :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="eleRoleText">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="eleTabIndexText">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TabIndex :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="eleTypeText">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type :</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_addMessage">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>391</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Add Message</string>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MessageId :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="msgIdText">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="addMsgBtn">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>110</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Message</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>isAccTextSame</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="locMsgText">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="accMsgText">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Loc :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Acc :</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>openDialog</tabstop>
  <tabstop>createNewInter</tabstop>
  <tabstop>jsViewNextLoadCheckBox</tabstop>
  <tabstop>jsViewList</tabstop>
  <tabstop>templateNextLoadCheckBox</tabstop>
  <tabstop>templateList</tabstop>
  <tabstop>cssList</tabstop>
  <tabstop>cssNextLoadCheckBox</tabstop>
  <tabstop>locTreeWidget</tabstop>
  <tabstop>screenNameText</tabstop>
  <tabstop>screenNameText_2</tabstop>
  <tabstop>addScreenBtn</tabstop>
  <tabstop>eleNameText</tabstop>
  <tabstop>eleAccIdText</tabstop>
  <tabstop>eleTypeText</tabstop>
  <tabstop>eleRoleText</tabstop>
  <tabstop>eleTabIndexText</tabstop>
  <tabstop>addEleBtn</tabstop>
  <tabstop>msgIdText</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>locMsgText</tabstop>
  <tabstop>accMsgText</tabstop>
  <tabstop>addMsgBtn</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
